mantle/kola/harness.go

```diff
@@ -733,11 +732,11 @@ func registerTestDir(dir, testprefix string, children []os.FileInfo) error {
 			}
 			userdata = conf.Ignition(string(v))
 		} else if isreg && (c.Name() == "config.bu" || c.Name() == "config.fcc") {
-			b, err := exec.Command("fcct", fpath).Output()
+			v, err := ioutil.ReadFile(filepath.Join(dir, c.Name()))
```

Comment:
It's less exact but technically we could always group the Ignition & Butane/FCCT sections together and use `conf.Unknown`'s detection logic no? Could maybe even loosen it to `config*` as the qualifier for a config object if we go that route.

Reply:
Yeah, I thought about that.  Since we already have separate command-line options to distinguish the config types, I'm inclined to keep using those, rather than combining them and expecting the code to guess.